src/renderer/drawers/OperationDetails/NFTOperationDetails.js

```diff
@@ -0,0 +1,89 @@
+// @flow
+
+import React, { useMemo } from "react";
+import { useTranslation } from "react-i18next";
+import {
+  OpDetailsSection,
+  OpDetailsTitle,
+  GradientHover,
+  OpDetailsData,
+  TextEllipsis,
+  HashContainer,
+} from "~/renderer/drawers/OperationDetails/styledComponents";
+import type { Operation } from "@ledgerhq/live-common/lib/types";
+import Image from "~/renderer/screens/nft/Image";
+import Box from "~/renderer/components/Box";
+import { nftsFromOperations } from "@ledgerhq/live-common/lib/nft/helpers";
+import { useNftMetadata } from "@ledgerhq/live-common/lib/nft/NftMetadataProvider";
+import CopyWithFeedback from "~/renderer/components/CopyWithFeedback";
+import Skeleton from "~/renderer/screens/nft/Skeleton";
+import { centerEllipsis } from "~/renderer/styles/helpers";
+
+const NFTOperationDetails = ({ operation }: { operation: Operation }) => {
+  const { t } = useTranslation();
+  const operations = useMemo(() => [operation], [operation]);
+  const nfts = nftsFromOperations(operations);
+  const { status, metadata } = useNftMetadata(nfts[0]?.collection?.contract, nfts[0]?.tokenId);
+  const show = useMemo(() => status !== "loaded", [status]);
+
+  return !nfts[0] ? null : (
```

Comment:
Because it's a single nft but the hook i was provided returns "nfts plural for operations" hence the hacky memoized operation array and nfts array when I only need one.